proto/nodenetworkservice/3.302.0.744/NodeNetworkService.pb.go

```diff
@@ -627,19 +628,22 @@ func file_NodeNetworkService_proto_rawDescGZIP() []byte {
 	return file_NodeNetworkService_proto_rawDescData
 }
 
-var file_NodeNetworkService_proto_enumTypes = make([]protoimpl.EnumInfo, 1)
-var file_NodeNetworkService_proto_msgTypes = make([]protoimpl.MessageInfo, 8)
-var file_NodeNetworkService_proto_goTypes = []interface{}{
-	(RequestType)(0),                             // 0: azure.nodenetworkservice.RequestType
-	(*ConfigureNetworkingRequest)(nil),           // 1: azure.nodenetworkservice.ConfigureNetworkingRequest
-	(*ConfigureNetworkingResponse)(nil),          // 2: azure.nodenetworkservice.ConfigureNetworkingResponse
-	(*PingNodeNetworkServiceRequest)(nil),        // 3: azure.nodenetworkservice.PingNodeNetworkServiceRequest
-	(*PingNodeNetworkServiceResponse)(nil),       // 4: azure.nodenetworkservice.PingNodeNetworkServiceResponse
-	(*ConfigureContainerNetworkingRequest)(nil),  // 5: azure.nodenetworkservice.ConfigureContainerNetworkingRequest
-	(*ConfigureContainerNetworkingResponse)(nil), // 6: azure.nodenetworkservice.ConfigureContainerNetworkingResponse
-	(*ContainerIPAddress)(nil),                   // 7: azure.nodenetworkservice.ContainerIPAddress
-	(*ContainerNetworkInterface)(nil),            // 8: azure.nodenetworkservice.ContainerNetworkInterface
-}
+var (
```

Comment:
Aren't .pb.go files auto generated ? These changes can get overwritten on any new updates to the proto files right ?

Reply:
dropped those from this change, good call